### PR TITLE
Switch to the PAE defined in the new DSSEv1 spec.

### PR DESCRIPTION
**Pull request by dlorenc**
*Please fill in the fields below to submit a pull request.  The more
information that is provided, the better.*

**Fixes issue #**:
N/A

**Description of pull request**:

The DDSE v1 spec changes the PAE, this updates to match that.

**Please verify and check that the pull request fulfills the following
requirements**:

- [X] Tests have been added for the bug fix or new feature
- [ ] Docs have been added for the bug fix or new feature

### CODE REVIEW

**Comment by dlorenc** [comment]
cc @kommendorkapten FYI.

**Review by shibumi — APPROVED** [review]

File: pkg/ssl/sign.go [hunk 1]
```diff
@@ -47,31 +46,11 @@ type Signature struct {
...
-		}
-
-		if _, err = buf.Write(b); err != nil {
-			return nil, err
-		}
-	}
-
-	return buf.Bytes(), nil
+func PAE(payloadType, payload string) []byte {
+	return []byte(fmt.Sprintf("DSSEv1 %d %s %d %s", len(payloadType), payloadType, len(payload), payload))
```
Comment (adityasaky) [inline pkg/ssl/sign.go:53]: Line length > 80

**Review by adityasaky — APPROVED** [review]
Minor nit, but LGTM! Thanks, @dlorenc :smile:

File: pkg/ssl/sign.go (hunk 1 above)
Comment (dlorenc) [reply pkg/ssl/sign.go]: Fixed!

Comment (adityasaky) [reply pkg/ssl/sign.go]: Thanks!

**Comment by dlorenc** [comment]
Thanks!

**Comment by adityasaky** [comment]
Do you need this in a release? We can probably do a 0.2.1? Or via #110.

**Comment by dlorenc** [comment]
> Do you need this in a release? We can probably do a 0.2.1? Or via #110.

I don't need it in one, I've been using exact commits anyway.

**Pull request merged**